.circleci/config.yml

```diff
@@ -92,7 +92,22 @@ jobs:
       - attach_workspace:
           at: ~/answers
       - run: ./.circleci/run_acceptance.sh
-        
+  # verify all translations are present
+  translation_test:
+    docker:
+      - image: circleci/node:14.5
+    working_directory: ~/answers
+    steps:
+      - setup-workspace
+      - attach_workspace:
+          at: ~/answers
+      - run: 
+          name: Verify translation files
+          command: |
+            npm run extract-translations
+            sudo apt-get install gettext
+            cd conf/i18n/translations
+            find . -type f -name "*.po" | xargs -n1 -I {{po_file}} msgcmp {{po_file}} messages.pot
```

Comment:
would it be a little clearer what's happening if we just did a for loop through the files in the translations folder? something like 
```sh
FILES=/path/to/*
for f in $FILES
```
happy with what we have right now as well it's very cool guy so up to you

Reply:
Updated to make this clearer!